packages/interbit-ui-components/webpack.config.js

```diff
@@ -26,22 +27,37 @@ const config = {
         loader: 'babel-loader',
         exclude: /(node_modules|bower_components)/
       },
-      {
-        test: /\.css$/,
-        loader: 'style-loader!css-loader?modules'
-      },
       {
         test: /\.(png|svg)$/,
         use: [
           {
             loader: 'url-loader',
             options: {
-              name: '[name][md5:hash].[ext]',
-              outputPath: 'assets/',
-              publicPath: '/assets/'
+              limit: 25000,
+              fallback: 'file-loader',
+              // Options for the fallback
```

Comment:
Leftover comment

Reply:
I think this is a helpful comment. The options that follow are for the file-loader and not the url-loader. They get passed through to the file-loader if the asset file is larger than the set limit.